assets/src/edit-story/app/story/actions/useSaveStory.js

```diff
@@ -74,19 +73,21 @@ function useSaveStory({ storyId, pages, story, updateStory }) {
         ...getStoryPropsToSave({ story, pages, metadata, flags }),
         ...props,
       })
-        .then((post) => {
+        .then((data) => {
           const properties = {
```

Comment:
```suggestion
          const { status, slug, link, featured_media_url: featuredMediaUrl, preview_link: previewLink, edit_link: editLink, embed_post_link: embedPostLink } = data;
         const properties = {
            status,
            slug,
            link,
            featuredMediaUrl,
            previewLink,
            editLink,
            embedPostLink
         };
```

This feel cleaner.

Reply:
Can't do this as-is since `editLink` is already declared on line 58 above.

I think the current way of writing is not too bad. No strong opinion though.